data/bootstrap.py

```diff
@@ -96,6 +98,14 @@
 
     shapefiles.append(shapefile)
 
+WIKIDATA_BASE_QUERY = 'https://query.wikidata.org/sparql'
```

Comment:
Can we cache the results and include in the asset bundle? Their Sparql endpoint is fairly reliable, but we shouldn't needlessly hit it again and again.

Reply:
Yup, that's what this is doing. The queries get run once and the results converted to a SQL file when the `shapefiles.tar.gz` bundle is being built, and any tile builds will just use the static SQL file from the bundle.